apps/frontend/src/pages/projects/libs/components/project-card/project-card.tsx

```diff
@@ -31,11 +37,33 @@ const ProjectCard = ({
 		onDelete(project);
 	}, [onDelete, project]);
 
+	const currentDate = new Date();
+	const lastActivityDate = project.lastActivityDate
+		? new Date(project.lastActivityDate)
+		: null;
+
+	const daysDifference = lastActivityDate
+		? getDifferenceInDays(currentDate, lastActivityDate)
+		: null;
+
+	const lastUpdateLabel = lastActivityDate
+		? getRelativeDate(lastActivityDate, currentDate)
+		: null;
+
+	const colorStatus =
+		typeof daysDifference === "number"
+			? getActivityIndicatorStatus(daysDifference)
+			: null;
```

Comment:
Can't we move all the calculations to a some sort of helper and just return the object of required variables?

Reply:
@liza-veis should I create `calculateActivityData` helper in projects module?

Reply:
let's leave this, this doesn't look like a logic that we should group as for me. All of this can be re-used separately later by the component